AI-and-Analytics/Getting-Started-Samples/LPOT-Sample-for-Tensorflow/sample.json

```diff
@@ -2,7 +2,7 @@
  "guid": "82e7612f-2810-4d12-9c75-c17fcbb946fa",
  "name": "Intel® Low Precision Optimization Tool Tensorflow Getting Started",
  "categories": ["Toolkit/oneAPI AI And Analytics/AI Getting Started Samples"],
- "description": "This sample illustrates how to run LPOT to quantize the FP32 model trained by Keras on Tensorflow to INT8 model to speed up the inference.",
+ "description": "This sample illustrates how to run Intel® LPOT to quantize the FP32 model trained by Keras on Tensorflow to INT8 model to speed up the inference.",
```

Comment:
Rich text supported in .json files?

Reply:
yes it works fine. Json isnt the issue, it will be how the individual sample browsers consume